Modal feature

kinginthenorthcodez opened the pull request:
## Modal feature
- Add Dynamic content for project sections
- Add Dynamic Modal for projects

warm regards
@aimalamiri 
@kinginthenorthcodez

kinginthenorthcodez committed:
Add Modal

Add modal assets

kinginthenorthcodez committed:
Add close btn for modal

aimalamiri committed:
Add projects data object

aimalamiri committed:
Add project modal script

aimalamiri committed:
Fix image size

kinginthenorthcodez committed:
Update script.js

kinginthenorthcodez committed:
Fix ejslint errors

kinginthenorthcodez committed:
Fix ejslint errors

kinginthenorthcodez committed:
Update codebase style.css script.js

kinginthenorthcodez committed:
Fix ESlint errorss

kinginthenorthcodez committed:
Fix ESlint errorss

omar-muhamad commented on assets/css/style.css:
```diff
@@ -459,9 +489,154 @@ footer .email {
+  margin: 0 12px;
+}
+
+.modal-body {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.modal-list {
+  display: flex;
+  list-style: none;
+  width: 245px;
+  height: 48px;
+  margin: 0  auto;
+  align-items: center;
+}
```
- [x] Kindly add the correct margin value between the project description and the list of technologies like The Figma design.

- [x] I can see the list of technologies overflowing from the popup container and make your site side-scrolling. Please fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/162232529-46eb860b-91ea-4ad1-8637-1c0df0595e77.png)  | ![chrome-capture-2022-3-7](https://user-images.githubusercontent.com/68796232/162232374-e3b051b5-4b56-45a5-a28e-6f2d0e9213bf.gif)  |

kinginthenorthcodez replied:
- Fixed margin value between the project description and the list of technologies 👍🏽 
- Fixed list of technologies overflowing from the popup container and  site side-scrolling 👍🏽

omar-muhamad reviewed (changes requested):
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.

![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

kinginthenorthcodez committed:
Review Required Changes

Fixed margin issue between modal-description and list of technologies
fixed overflow on pop technologies and site side-scrolling

kinginthenorthcodez committed:
fixed Eslint error

zainsadaqat reviewed (approved):
# Hello @kinginthenorthcodez 😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 

<img src="https://media3.giphy.com/media/3KC2jD2QcBOSc/giphy.gif"/>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kinginthenorthcodez commented:
Merge #10 into main
